docs/io/fits/usage/table.rst

```diff
@@ -365,3 +365,131 @@ to the HDU's data attribute:
 
 In a future version of Astropy table creation will be simplified and this
 process won't be necessary.
+
+.. _fits_time_column:
+
+FITS time column
+----------------
+
+To create FITS columns which adhere to the FITS Time standard, we need to take
+into account the following important points stated in the standard.
+
+The strategy used to store `~astropy.time.Time` columns in FITS tables is to use
```

Comment:
At this point, I was a little confused if I are supposed to do these steps as a user or if the code does it. I would change the introductory sentence and say something like:

The FITS standard defines the keywords and formats used to write timing information to fits files and in many cases astropy can just write an astropy table in this format 

... simple example here, max 5 lines...

However, the FITS standard has some limitations and there are cases where not all information in an astropy table can be mapped perfectly into the FITS standard compliant file. Also, the FITS standard in some cases allows choices where e.g. times can be written with an absolute value or relative to a reference time. To help the user understand how the astropy code deals with these situations, the following text describes the approach that astropy takes in some details.